src/language-js/printer-estree.js

```diff
@@ -553,7 +553,11 @@ function printPathNoParens(path, options, print, args) {
       );
     case "BinaryExpression":
     case "LogicalExpression":
-    case "NGPipeExpression": {
+    case "NGPipeExpression":
+    case "TSAsExpression": {
+      const leftNodeName = n.type === "TSAsExpression" ? "expression" : "left";
+      const rightNodeName =
+        n.type === "TSAsExpression" ? "typeAnnotation" : "right";
```

Comment:
Let's extract this to a function so that it would be called like this:
```js
const { leftNodeName, rightNodeName, operator } = getBinaryishNodeNames(node);
```

Reply:
Yes, I was just trying to do that:smile: